pkg/inventory/clusterrolebinding_test.go

```diff
@@ -0,0 +1,75 @@
+package inventory
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	v1beta1 "k8s.io/api/rbac/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestClusterRoleBindingInventory(t *testing.T) {
+	toCreate := v1beta1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "to-create",
+		},
+	}
+	toUpdate := v1beta1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "to-update",
+		},
+		Subjects: []v1beta1.Subject{{
+			Name: "serviceaccount1",
+		}},
+	}
+	updated := v1beta1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        "to-update",
+			Annotations: map[string]string{"gopher": "jaeger"},
+			Labels:      map[string]string{"gopher": "jaeger"},
+		},
+		Subjects: []v1beta1.Subject{{
+			Name: "serviceaccount2",
+		}},
+	}
+	toDelete := v1beta1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "to-delete",
+		},
+	}
+
+	existing := []v1beta1.ClusterRoleBinding{toUpdate, toDelete}
+	desired := []v1beta1.ClusterRoleBinding{updated, toCreate}
+
+	inv := ForClusterRoleBindings(existing, desired)
+	assert.Len(t, inv.Create, 1)
+	assert.Equal(t, "to-create", inv.Create[0].Name)
+
+	assert.Len(t, inv.Update, 1)
+	assert.Equal(t, "to-update", inv.Update[0].Name)
+	assert.Equal(t, "serviceaccount2", inv.Update[0].Subjects[0].Name)
+
+	assert.Len(t, inv.Delete, 1)
+	assert.Equal(t, "to-delete", inv.Delete[0].Name)
+}
+
+func TestClusterRoleBindingInventoryWithSameNameInstances(t *testing.T) {
```

Comment:
Question - how would this happen - having two with same name?

Reply:
Oops, copy/paste :)